src/features/variables/variablesSlice.ts

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from "@reduxjs/toolkit";
```

Comment:
This file here is essentially the same as `src/features/channels/channelsSlice.ts`. I've just replaced the variable name and type.